src/megatron/bridge/recipes/qwen/qwen3_480b_a35b.py

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List, Optional, Union
+
+import torch
+from megatron.core.distributed import DistributedDataParallelConfig
+
+from megatron.bridge.models.qwen import Qwen3MoEModelProvider480B_A35B
+from megatron.bridge.recipes.utils.dataset_utils import get_blend_fields_from_data_paths
+from megatron.bridge.recipes.utils.optimizer_utils import distributed_fused_adam_with_cosine_annealing
+from megatron.bridge.training.comm_overlap import CommOverlapConfig
+from megatron.bridge.training.config import (
+    CheckpointConfig,
+    ConfigContainer,
+    GPTDatasetConfig,
+    LoggerConfig,
+    RNGConfig,
+    TokenizerConfig,
+    TrainingConfig,
+)
+from megatron.bridge.training.mixed_precision import MixedPrecisionConfig
+
+
+def model_config(
+    tensor_parallelism: int = 4,
+    pipeline_parallelism: int = 32,
+    pipeline_parallelism_dtype: Optional[torch.dtype] = torch.bfloat16,
+    virtual_pipeline_parallelism: Optional[int] = None,
+    context_parallelism: int = 2,
+    expert_parallelism: Optional[int] = 8,
+    sequence_parallelism: bool = True,
+) -> Qwen3MoEModelProvider480B_A35B:
+    """
+    Configure the Qwen3 480B-A35B MoE model.
+
+    Args:
+        tensor_parallelism (int): Degree of tensor model parallelism.
+        pipeline_parallelism (int): Degree of pipeline model parallelism.
+        pipeline_parallelism_dtype (Optional[torch.dtype]): Data type for pipeline parallelism.
+        virtual_pipeline_parallelism (Optional[int]): Size of virtual pipeline parallelism.
+        context_parallelism (int): Degree of context parallelism.
+        expert_parallelism (Optional[int]): Degree of expert parallelism for MoE.
+        sequence_parallelism (bool): Whether to use sequence parallelism.
+
+    Returns:
+        Qwen3MoEModelProvider235B_A22B: Configuration for the Qwen3 235B-A22B MoE model.
+    """
+    model_cfg = Qwen3MoEModelProvider480B_A35B(
+        tensor_model_parallel_size=tensor_parallelism,
+        pipeline_model_parallel_size=pipeline_parallelism,
+        pipeline_dtype=pipeline_parallelism_dtype,
+        virtual_pipeline_model_parallel_size=virtual_pipeline_parallelism,
+        context_parallel_size=context_parallelism,
+        expert_model_parallel_size=expert_parallelism,
```

Comment:
```suggestion
        expert_model_parallel_size=expert_parallelism,
        expert_tensor_parallel_size=1,
```
this is missing from the existing qwen3 MOE recipes but is being fixed here: https://github.com/NVIDIA-NeMo/Megatron-Bridge/pull/456